### PR TITLE
Fix SPOUTPLUGIN-55 by allowing auto sprint to be disabled at a server level

### PR DESCRIPTION
**Pull request by md-5**
Title says it all. Spoutcraft PR also opened.

### CODE REVIEW

**Comment by Spouty** [comment]
**[AUTO-GENERATED COMMENT]**

This pull request has only **_1 commit**_, is **_signed off**_, and **_compiles**_.
Thanks for following the [pull request guidelines](http://spout.in/prguide) and for your contribution!
A [team member](http://spout.org/team) will review this pull request when time allows.

Maven artifacts packaged from this pull request: 
- [spoutplugin-dev-SNAPSHOT.jar](http://pr.spouty.org/files/?file=3306187105788465/spoutplugin-dev-SNAPSHOT.jar)
- [original-spoutplugin-dev-SNAPSHOT.jar](http://pr.spouty.org/files/?file=3306187105788465/original-spoutplugin-dev-SNAPSHOT.jar)

File: src/main/java/org/getspout/spoutapi/packet/PacketAllowVisualCheats.java [hunk 1]
```diff
@@ -33,12 +33,13 @@
 	private boolean coords = false;
 	private boolean entitylabel = false;
 	private boolean voidfog = false;
+	private boolean sprint = false;
 
 	public PacketAllowVisualCheats() {
```
Comment (ZNickq) [inline src/main/java/org/getspout/spoutapi/packet/PacketAllowVisualCheats.java:38]: You should bump up the getVersion method if you change the packet ;)

**Comment by Afforess** [comment]
Haha. No.

**Comment by md-5** [comment]
Care to elaborate on why? Someone opened a perfectly legitimate feature request and I fulfilled that request.
http://issues.spout.org/browse/SPOUTPLUGIN-55
If there are any issues with my implementation (thanks @ZNickq) then I would prefer constructive feedback on these issues so that they can be rectified in order that this pull may proceed further.

**Comment by Afforess** [comment]
Sprinting isn't cheating. I don't allow admins to disable essentially vanilla features.

**Comment by mbax** [comment]
Rendering blocks is a vanilla feature, xray texture packs are a cheat.  Flying is a vanilla feature. Enabling that client-side when the server doesn't want it is a cheat.  I think this would be a useful feature, assuming that it's only preventing the cheat-functionality of sprinting and not disabling all sprinting.

**Comment by Afforess** [comment]
Technically there is no cheat functionality in sprinting anymore. It disables when you don't have the stamina to sprint, it does not allow reverse sprinting. It stops when you run into a wall.

**Comment by md-5** [comment]
Just to clarify this does not disable sprinting in the vanilla context, but rather Spoutcraft's new sprint toggle, or auto sprint if you will.
If the issues raised in the above mentioned ticket have been resolved and the reporter is satisfied that the issue has been resolved, then the above ticket as well as this pull request can be closed / remain closed.

**Pull request closed without merging**